src/node/interfaces.cpp

```diff
@@ -168,29 +168,25 @@ class NodeImpl : public Node
     bool getNodesStats(NodesStats& stats) override
```

Comment:
Couldn't we change this function to return `NodeStats` instead of `bool`. Seems odd the first thing we do here is `clear` the `stats`.

Reply:
Good observation! Done.